data.sql

```diff
@@ -1,28 +1,27 @@
 \c education 
-
 DROP TABLE IF EXISTS users;
 
+DROP TABLE IF EXISTS schools;
+
+CREATE TABLE schools (
+    id integer PRIMARY KEY,
+    name text,
+    district text,
+    state text
+);
+
 CREATE TABLE users (
-    username text PRIMARY KEY,
-    password text NOT NULL,
+    email text PRIMARY KEY,
```

Comment:
I would not make email a primary key - a user may change their email. I would suggest a guid or an int- something that is constant

Reply:
Thanks for catching that. I had it planned as an int in my diagram, I'll change it now.

Reply:
Fixed